spec/std/http/chunked_content_spec.cr

```diff
@@ -9,7 +9,7 @@ describe HTTP::ChunkedContent do
     bytes_read = content.read(bytes.to_slice)
     bytes_read.should eq(4)
     String.new(bytes.to_slice).should eq("123\n")
-    mem.pos.should eq(7) # only this chunk was read
+    mem.pos.should eq(9) # only this chunk was read
```

Comment:
This spec shouldn't change, see https://github.com/crystal-lang/crystal/issues/3270. There's even a nice comment linking to this PR, right next to code you changed.